01.fizzbuzz/FizzBuzz.rb

```diff
@@ -0,0 +1,11 @@
+1.upto(20) do |num|
+  if num % 3 == 0 && num % 5 == 0
+    puts("FizzBuzz")
```

Comment:
```suggestion
    puts "FizzBuzz"
```

のように`()`を付けない書き方をよく見かけますが、MUSTではないのでこのままでも構いません。